{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/tests.py

```diff
@@ -8,6 +9,10 @@ def test_base(self):
         response = self.client.get(reverse('home'))
         assert b'csrfmiddlewaretoken' in response.content
 
+    def test_fe_filter(self):
+        response = self.client.get(reverse('home'))
+        assert 'DEBUG setting: {0}'.format(settings.DEBUG) in response.content
```

Comment:
I think you need a `b` like in https://github.com/mozilla/sugardough/pull/98/files#diff-3fed87ec34a6808ba36f7d63ed41b42cR10 to make this work in python3